lib/rubocop/cop/style/unneeded_percent_q.rb

```diff
@@ -14,6 +14,7 @@ def on_dstr(node)
         end
 
         def on_str(node)
+          return unless standalone_string?(node)
```

Comment:
I was surprised by this as well. Given the example string `"%q(a)#{b}"`, this cop was called twice. Once for `dstr` with `"%q(a)#{b}"`, and a second time for `str` with the substring `"%q(a)"`. It seemed weird to me that this was the behavior. That is why I had to add in the check to see if the string is a substring.